Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -78,8 +88,15 @@ var TouchableWithoutFeedback = React.createClass({
   },
 
   render: function(): ReactElement {
-    // Note(vjeux): use cloneWithProps once React has been upgraded
-    var child = onlyChild(this.props.children);
+    var child = cloneWithProps(
```

Comment:
Actually, can you use React.cloneElement, cloneWithProps isn't working because it blows away the key